forge/ee/db/models/SAMLProvider.js

```diff
@@ -0,0 +1,96 @@
+const { buildPaginationSearchClause } = require('../../../db/utils')
+const { DataTypes } = require('sequelize')
+
+module.exports = {
+    name: 'SAMLProvider',
+    schema: {
+        name: { type: DataTypes.STRING, allowNull: false },
+        domainFilter: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            set (value) {
+                value = (value || '').toLowerCase()
+                if (value[0] !== '@') {
+                    value = '@' + value
+                }
+                this.setDataValue('domainFilter', value)
+            }
+        },
+        active: { type: DataTypes.BOOLEAN, defaultValue: true },
+        acsURL: { type: DataTypes.VIRTUAL, get () { return process.env.FLOWFORGE_BASE_URL + '/ee/sso/login/callback' } },
+        entityID: { type: DataTypes.VIRTUAL, get () { return process.env.FLOWFORGE_BASE_URL + '/ee/sso/entity/' + this.hashid } },
+        options: {
+            type: DataTypes.TEXT,
```

Comment:
There is a DataTypes.JSON that will do the get/set stuff for you, it's good on all the DBs we use (and are likely to use)

https://sequelize.org/docs/v6/other-topics/other-data-types/#json-sqlite-mysql-mariadb-oracle-and-postgresql-only